bin/oxfordshire/send-rdi-emails

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env perl
```

Comment:
I think I'd prefer leaving that for such a time as that might (hopefully never) happen ;) It's not a very long script, the main bulk is already factored out generating the RDI, other than that it's only a small loop.